### PR TITLE
Alpha Support for SQL/JDBC

### PR DESCRIPTION
**Pull request by alainbodiguel**
Partial implementation of SQL/JDBC support

### CODE REVIEW

**Review by sfalquier — APPROVED** [review]
For a first draft, LGTM

File: arlas-core/src/main/java/io/arlas/server/app/ArlasBaseConfiguration.java [hunk 1]
```diff
@@ -51,9 +57,7 @@
     public static final String FLATTEN_CHAR = "_";
 
     public void check() throws ArlasConfigurationException {
-        elasticConfiguration.check();
```
[the remark this answers is not in the document]
Comment (alainbodiguel) [reply arlas-core/src/main/java/io/arlas/server/app/ArlasBaseConfiguration.java:54]: It's been moved to `ElasticDatabaseToolsFactory`

File: arlas-core/src/main/java/io/arlas/server/utils/ParamsParser.java [hunk 2]
```diff
@@ -520,6 +522,17 @@ public static Page getPage(IntParam size, IntParam from, String sort, String aft
...
+        Pair<Double, Double> latLon = getGeoSortParamsAsLatLon(geoSort);
+        return new GeoPoint(latLon.getLeft(), latLon.getRight());
+    }
+
+    /**
+     *
+     * @param geoSort
+     * @return Pair(lat, lon)
+     * @throws ArlasException
+     */
```
[the remark this answers is not in the document]
Comment (alainbodiguel) [reply arlas-core/src/main/java/io/arlas/server/utils/ParamsParser.java:534]: Fixed

File: conf/configuration.yaml [hunk 3]
```diff
@@ -173,4 +176,28 @@ arlas_cors:
   exposed_headers: ${ARLAS_CORS_EXPOSED_HEADERS:-"Content-Type,Authorization,X-Requested-With,Content-Length,Accept,Origin,Location,WWW-Authenticate"}
 
 arlas_database_factory_class: ${ARLAS_DATABASE_FACTORY_CLASS:-io.arlas.server.impl.ElasticDatabaseToolsFactory}
+#arlas_database_factory_class: ${ARLAS_DATABASE_FACTORY_CLASS:-io.arlas.server.impl.PostgisDatabaseToolsFactory}
 arlas_cache_factory_class: ${ARLAS_CACHE_FACTORY_CLASS:-io.arlas.server.impl.HazelcastCacheFactory}
+
+database:
```
[the remark this answers is not in the document]
Comment (alainbodiguel) [reply conf/configuration.yaml:182]: Done

**Review by mbarbet — APPROVED** [review]

**Review by MohamedHamouGisaia — APPROVED** [review]

File: arlas-core/src/main/resources/io/arlas/server/impl/jdbi/dao/CollectionReferenceDao/describeGeometryType.sql [hunk 4]
```diff
@@ -0,0 +1,13 @@
+SELECT
+    f_table_name AS table_name,
+    f_geometry_column AS column_name,
+--    type AS data_type
+    CASE
+        WHEN (type = 'POINT') THEN 'GEO_POINT'
+        WHEN (type = 'MULTIPOLYGON') THEN 'GEO_SHAPE'
```
[the remark this answers is not in the document]
Comment (alainbodiguel) [reply arlas-core/src/main/resources/io/arlas/server/impl/jdbi/dao/CollectionReferenceDao/describeGeometryType.sql:7]: Fixed

**Review by MohamedHamouGisaia — APPROVED** [review]

**Pull request merged**